test/examples/uct_hello_world.c

```diff
@@ -597,20 +597,20 @@ int main(int argc, char **argv)
         }
     }
 
-    status = sendrecv(oob_sock, own_dev, if_info.iface_attr.device_addr_len,
-                      (void **)&peer_dev);
+    status = (ucs_status_t)sendrecv(oob_sock, own_dev, if_info.iface_attr.device_addr_len,
```

Comment:
sendrecv returns int of 0 or -1
so need to use another variable (eg "int ret") instead of status

Reply:
fixed